easybuild/easyblocks/c/clang.py

```diff
@@ -166,13 +166,24 @@ def configure_step(self):
             self.llvm_obj_dir_stage3 = os.path.join(self.builddir, 'llvm.obj.3')
 
         if LooseVersion(self.version) >= LooseVersion('3.3'):
```

Comment:
Yeah, great idea no? :laughing: 

I would change the warning a bit (add: unless strict or something).